src/main/kotlin/com/exactpro/th2/service/generator/protoc/generator/FileSpecImpl.kt

```diff
@@ -13,17 +13,10 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.exactpro.th2.service.annotation;
+package com.exactpro.th2.service.generator.protoc.generator
```

Comment:
Why `generator.protoc.generator`? I think one `generator` in the package name will be enough

Reply:
`generator.protoc` - main package for protoc plugin. `generator.protoc.generator` - for implementation for interface `Generator`. May be rename it to `generator.protoc.impl`?

Reply:
It would be great) The current package name is a bit strange. But we should check all other places in the project and do the same.

Reply:
Or we can split generators by language they generate wrappes for.
So it will be `generator.protoc.java` and `generator.protoc.python`

Reply:
Now in package `generator.protoc.generator` already have `java` and `python`. I will remove second `generator` in package name.